.github/workflows/cffconvert.yml

```
@@ -0,0 +1,24 @@
+name: cffconvert
+
+on:
+  push:
+    paths:
+      - CITATION.cff
```

Currently, this workflow will not be triggered on pull requests that come from forks.

I think the behavior we want is this:
1. If the `CITATION.cff` file was modified), and this is a pull request, and the base (target) branch of the pull request is either `master` or `release-*`, then run the workflow.
2. If the `CITATION.cff` file was modified), and this is a push to either `master` or `release-*`, then run the workflow.
3. Else, do not run the workflow.

I think that the following suggestion implements the above behavior, although it would be good for someone else to double-check.

```suggestion
on:
  push:
    branches:
      - 'master'
      - 'release-*'
    paths:
      - CITATION.cff
  pull_request:
    branches:
      - 'master'
      - 'release-*'
    paths:
      - CITATION.cff
```